packages/l/libchewing/package.yml

```diff
@@ -1,19 +1,22 @@
 name       : libchewing
-version    : 0.5.1
-release    : 2
+version    : 0.8.3
+release    : 3
 source     :
-    - https://github.com/chewing/libchewing/releases/download/v0.5.1/libchewing-0.5.1.tar.bz2 : 9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b
+    - https://github.com/chewing/libchewing/releases/download/v0.8.3/libchewing-0.8.3.tar.zst : 6c8734eb3e5bbb7e9ba407d1315ffdaa8770e4c21fca835fb045329ef7fd3a1c
+homepage   : https://chewing.im/
 license    : LGPL-2.1-only
 component  : desktop.library
 summary    : Chewing is an intelligent phonetic (Zhuyin/Bopomofo) input method
 description: |
     Chewing is an intelligent phonetic (Zhuyin/Bopomofo) input method
+networking : yes
 builddeps  :
     - pkgconfig(sqlite3)
+    - git
+    - rust
```

Comment:
Does this have to be `rust-devel` now, since the binaries were moved?

Reply:
Not sure. cargo is in `rust` package. It doesn't seem to need all the others in `rust-devel` package